maintenance_plan/tests/test_maintenance_plan.py

```diff
@@ -145,7 +145,9 @@ def test_generate_requests(self):
             self.assertEqual(
                 fields.Date.to_date(req.schedule_date), request_date_schedule
             )
-            request_date_schedule = request_date_schedule + relativedelta(months=1)
+            request_date_schedule = request_date_schedule + relativedelta(
```

Comment:
Why putting relative dates instead of absolute ones?

Reply:
Maintenance plans are currently created using today's date.

Another possibility would be to change all tests to run on specific dates and check specific dates.

Reply:
You can use freezegun for having fixed dates.